contents/blog/how-we-built-email-onboarding.md

```diff
@@ -0,0 +1,132 @@
+---
+date: 2024-02-05
+title: "How we built our onboarding email flow (with actual performance data)"
+rootPage: /blog
+category: Inside PostHog
+author: ["joe-martin"]
+featuredImage: ../images/blog/blog_onboarding.jpg
+featuredImageType: full
+---
+
+Email is a powerful tool for getting users to engage with your product, but it can be difficult to know how the best companies structure their onboarding email flows. You end up having to sign up to a bunch of them, then waiting and letting your inbox get flooded. 
+
+Often, you end up getting awful sales rep outreach emails from them _months_ later too. That's important because it's probably the root cause for the conventional wisdom that developers do not respond well to email marketing. Turns out, they just don't respond well to _bad_ email marketing. 
```

Comment:
```suggestion
Conventional wisdom among marketers is that onboarding emails are a powerful tool for driving engagement. Convention wisdom among everyone else is that they're terrible and ineffective.

As always, the conventional wisdom isn't entirely accurate - but it can be hard to tell because there's no way to know what a anyones onboarding emails look like without signing up and letting them flood your inbox. Often that means getting flooded with awful outreach emails from sales reps. 
```

Better?

Reply:
```suggestion
Marketers believe onboarding emails are a powerful tool for driving engagement. Everyone else thinks they're annoying and ineffective. The truth, as always, is a little more complicated.

People, and developers in particular, hate onboarding campaigns because the bad ones are so memorable. And how would you know what "good" looks like without signing up and risking a flood of tedious outreach emails from sales reps?
```

Reply:
Based on your suggestion, but tweaked a bit.